clang/lib/Tooling/CommonOptionsParser.cpp

```diff
@@ -176,6 +176,11 @@ OPT_TYPE OPT_VAR(OPTION_NAME, __VA_ARGS__);
 
   SourcePathList = SourcePaths;
 #ifdef SYCLomatic_CUSTOMIZATION
+#ifndef _WIN32
+  if (std::string(argv[1]) == "--intercept-build") {
```

Comment:
how about make "-intercept-build" also works.

Reply:
Modified and tested to make `-intercept-build` and `intercept-build` to also work. Done.